client/src/js/services/receipts/ReceiptService.js

```diff
@@ -18,39 +18,54 @@ function ReceiptService($http, util) {
     HTML : 'html',
     JSON : 'json'
   };
-  
+
   service.invoice = invoice;
   service.patient = patient;
   service.renderers = renderers;
-  
+  service.patientRegistrations = patientRegistrations;
+
   /**
    * Fetch invoice report data from /reports/invoices/:uuid
    *
    * @param {String} uuid      Target invoice UUID to report on
    * @param {Object} options   Configuration options for the server generated
-   *                            report, this includes things like render target.
-   * @return {Promise}          Eventually returns report object from server
+   *                           report, this includes things like render target.
+   * @return {Promise}         Eventually returns report object from server
    */
   function invoice(uuid, options) {
     var route = '/reports/invoices/'.concat(uuid);
     var responseType = null;
 
-    if (options.render === renderers.PDF) {
+    if (options.renderer === renderers.PDF) {
       responseType = 'arraybuffer';
     }
     return $http.get(route, {params: options, responseType: responseType})
       .then(util.unwrapHttpResponse);
   }
-  
+
   function patient(uuid, options) {
     var route ='/reports/patient/'.concat(uuid);
     var responseType = null;
-    
-    if (options.render === renderers.PDF) { 
+
+    if (options.renderer === renderers.PDF) {
+      responseType = 'arraybuffer';
```

Comment:
Just a note to say that in the future we could wrap this funcitonality in a utility method - there are a lot of crucial constants to get wrong here. 

First thing that comes to mind:
```js
let params;
if (options.renderer === renderers.PDF) { 
  // returns 
  // { 
  //   params : options,
  //   responseType : 'arraybuffer'
  //  }
  params = utility.configurePDFParams(options);
}

$http.get(route, params);
```